lib/util/git/index.ts

```diff
@@ -1122,6 +1122,28 @@ export async function fetchBranch(
   }
 }
 
+export async function configureCredentialHelperStore(
+  endpoint: string,
+  token: string,
+): Promise<void> {
+  logger.debug("Configuring credential helper and git-credential file");
+  try {
+    await gitRetry(() => git.raw(['config', 'credential.helper', 'store']));
```

Comment:
Comment what this function does. Also I'm not sure if it needs a retry - if it fails once it will fail 3 times